lib/spdy-transport/protocol/spdy/parser.js

```diff
@@ -172,7 +173,7 @@ Parser.prototype.onSynHeadFrame = function onSynHeadFrame(type,
 
     if (!isPush) {
       callback(null, {
-        type: 'HEADERS',
+        type: 'SYN_STREAM',
```

Comment:
Got it, does that mean that it sends back a HEADERS type code instead of the SYN REPLY? Didn't notice that, or does this just happens on the parsing part.